app/models/workflow.rb

```diff
@@ -5,7 +5,8 @@ class Workflow < ApplicationRecord
   default_scope { order(:sequence => :asc) }
 
   belongs_to :template
-  has_many :requests, -> { order(:id => :asc) }, :inverse_of => :workflow
+  before_destroy :validate_deletable, :pretend => true
```

Comment:
Did you mean `:prepend => true` ?

https://stackoverflow.com/questions/49577447/rails-5-before-destroy-callback-with-prepend-or-without-fails

Reply:
Yes! 👍